lms/djangoapps/instructor/services.py

```diff
@@ -78,13 +76,60 @@ def delete_student_attempt(self, student_identifier, course_id, content_id, requ
             except (StudentModule.DoesNotExist, enrollment.sub_api.SubmissionError):
                 err_msg = (
```

Comment:
this looks like the same error message as above, curious if this could be made DRY

Reply:
Slight differences in the messages. They all have a common prefix, but a final message that indicates what the difference was. I can probably still refactor though. Nice idea.